tasks/src/test/java/webdriver/google_cloud/WebDriverGoogleCloudTest.java

```diff
@@ -0,0 +1,59 @@
+package webdriver.google_cloud;
+
+import static webdriver.google_cloud.constants.Constants.*;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import webdriver.google_cloud.page.GoogleCloudCalculatorPage;
+import webdriver.google_cloud.page.GoogleCloudHomePage;
+import webdriver.google_cloud.service.GoogleCloudService;
+
+public class WebDriverGoogleCloudTest {
+  private WebDriver driver;
+
+  @BeforeMethod(alwaysRun = true)
+  public void browserSetup() {
+    driver = new ChromeDriver();
+    driver.manage().window().maximize();
+    new GoogleCloudHomePage(driver)
+        .openPage();
+    new GoogleCloudService(driver)
+        .navigateToCalculatorPage()
+        .fillCalculatorForm();
+  }
+
+  @Test(description = "Verify that details for compute engine displaying correctly on Calculator page")
+  public void verifyDropdownValues() {
+    GoogleCloudCalculatorPage calculatorPage = new GoogleCloudCalculatorPage(driver);
+    SoftAssert softAssert = new SoftAssert();
+    softAssert.assertTrue(calculatorPage.getMachineClassText().equals(MACHINE_CLASS_REGULAR),
```

Comment:
assertTrue/assertFalse doesn't say much in case of failures. To check for equality you should use assertEquals assertion, e.g. `softAssert.assertEquals(calculatorPage.getMachineClassText(), MACHINE_CLASS_REGULAR, ERROR_MACHINE_CLASS);`

Please fix in all places